web/src/main/java/com/futurewei/alcor/web/restclient/NetworkConfigManagerRestClient.java

```diff
@@ -45,25 +45,49 @@
 
     @DurationStatistics
     public void createNodeInfo(NodeInfoJson message) throws Exception {
+        String ncmUri = message.getNodeInfo().getNcmUri();
+        if (ncmUri == null)
+            ncmUri = ncmManagerUrl;
```

Comment:
Maybe we want to rename "ncmManagerUrl" to "defaultNcmUrl"?

Reply:
Do you want me to make the change and push into this PR, or will you do it in your merge?

Reply:
Let us make it in this PR.